src/app/ApplicationTemplate.Presentation/Configuration/AppServicesConfiguration.cs

```diff
@@ -1,18 +1,10 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Reactive;
-using System.Reactive.Concurrency;
-using System.Reactive.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using ApplicationTemplate;
+﻿using System.Reactive.Concurrency;
 using ApplicationTemplate.Business;
 using ApplicationTemplate.Client;
 using ApplicationTemplate.Presentation;
 using MessageDialogService;
 using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Localization;
-using Microsoft.Extensions.Logging;
+using Xamarin.Essentials;
```

Comment:
Do we still need those namespaces as you seem to not using X.E anymore ?

Reply:
Currently removing those reference, should be done in the next commit.